draft-ietf-capport-architecture.xml

```diff
@@ -168,15 +168,15 @@
             A device MAY query this API at any time to determine whether the
             network is holding the device in a captive state.
          </li>
-        <li>End-user devices can be notified of captivity with Captive Portal
-            Signals in response to traffic. This notification works in response
+        <li>End-user devices can be signalled that they could be captive with Captive Portal
```

Comment:
Maybe take the active voice here:

A Captive Network (term?) could signal User Equipment to notify them in response to transmissions by the User Equipment.

Reply:
Hmmm. Yeah. We missed this when doing the terminology cleanup

Reply:
My only concern with the new phrasing is that we've dropped 'Captive Portal Signal'. However, I think that makes this point a good segue into the next one: this says that the UE can be signalled. The next describes what that signal means.